bundles/org.eclipse.swt/Eclipse SWT Browser/win32/org/eclipse/swt/browser/Edge.java

```diff
@@ -64,112 +74,131 @@ public WebViewEnvironment(ICoreWebView2Environment environment) {
 	boolean inNewWindow;
 	HashMap<Long, LocationEvent> navigations = new HashMap<>();
 
+	private String lastCustomText;
+
 	static {
-		NativeClearSessions = () -> {
-			ICoreWebView2CookieManager manager = getCookieManager();
-			if (manager == null) return;
+		URI_FOR_CUSTOM_TEXT_PAGE = setupAndGetLocationForCustomTextPage();
+		setupCookieAndSessionHandlers();
+	}
 
-			long[] ppv = new long[1];
-			int hr = callAndWait(ppv, completion -> manager.GetCookies(null, completion));
+private static void setupCookieAndSessionHandlers() {
```

Comment:
Can you please align the intendentation of the refactored methods with the existing indentation, so that not all this code is marked as modified? The formatting change makes reviewing and history lookups unnecessarily difficult.

Reply:
In the file itself the methods don't have any indentation while the static block does. If I add indentation to the method it will look different from the other methods.

Reply:
And I moved the code in the static block about cokkie and session to a method setupCookieAndSessionHandlers so that's why it appears moved.

Reply:
Okay, indentation is inconsistent even now, I see.
Taking another look: why do we need the static initializer at all? Why not have
```java
private static final URI URI_FOR_CUSTOM_TEXT_PAGE = createUriForCustomText();
```
and the generation in that method of the URI? Is there some technical limitation is do not see?

Reply:
Oh yeah. I just didn't think about it. Just made the change.